vaccine_feed_ingest/runners/us/carbon_health/normalize.py

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env python
+
+import datetime
+import os
+import pathlib
+import re
+import sys
+from typing import Any, List, Optional
+
+import orjson
+import us
+from vaccine_feed_ingest_schema import location as schema
+from vaccine_feed_ingest_schema.common import BaseModel
+
+from vaccine_feed_ingest.utils.log import getLogger
+from vaccine_feed_ingest.utils.normalize import normalize_zip
+
+# Performance optimization: skip validation in our pydantic models.
+#
+# This is safe to do because the ingestion framework will do the same
+# validation after running us.
+optimized = True
+if optimized:
+    BaseModel.create = BaseModel.construct
```

Comment:
This idea didn't quite work. I checked out your PR and optimized=False didn't run.

I made some modifications, and this way works both optimized and not:
```
OPTIMIZED = True


def _create_instance(cls, *args, **kwds):
    if OPTIMIZED:
        return cls.construct(*args, **kwds)
    else:
        return cls(*args, **kwds)


BaseModel.create = classmethod(_create_instance)
```

Reply:
Thank you. I've added this to the PR.